src/screens/LocalConfirmNotificationCodeScreen.tsx

```diff
@@ -89,35 +63,34 @@ class ConfirmNotificationCodeScreen extends PureComponent<Props, State> {
   };
 
   onConfirm = () => {
-    const { pin, createNotificationEmail, email } = this.props;
+    const { pin } = this.props;
+    const { onSuccess } = this.props.route.params;
 
     const { userCode } = this.state;
 
     const passedCode = pin === userCode;
 
     if (passedCode) {
-      createNotificationEmail(email, {
-        onSuccess: this.onSuccess,
-      });
+      onSuccess();
     } else {
       this.onError();
     }
   };
 
   render() {
     const { error, userCode, numberAttempt } = this.state;
-    const { email } = this.props.route.params;
+    const { children, title } = this.props.route.params;
```

Comment:
but only one screen goes to LocalConfirmNotificationCode so why the body is as param?

Reply:
We don't have this flow currently but the text would be different on update flow, so I thought it would be nice to parameter it
![Zrzut ekranu 2021-01-22 o 12 30 02](https://user-images.githubusercontent.com/67273035/105485726-9e4ee400-5cad-11eb-8f2c-80ae2931f86e.png)
![Zrzut ekranu 2021-01-22 o 12 30 20](https://user-images.githubusercontent.com/67273035/105485730-9f801100-5cad-11eb-8713-d611da898774.png)